c_src/erocksdb.cc

```diff
@@ -1161,12 +1354,102 @@ checkpoint(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[])
 }   // checkpoint
 
 
+ERL_NIF_TERM
+create_column_family(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[])
+{
+    ReferencePtr<DbObject> db_ptr;
+    const ERL_NIF_TERM& handle_ref = argv[0];
+    db_ptr.assign(DbObject::RetrieveDbObject(env, handle_ref));
+
+    if(NULL==db_ptr.get())
+    {
+        return enif_make_badarg(env);
+    }
+
+    char cf_name[4096];
```

Comment:
strictly speaking,
as there is no limit about the length of a column family name,
we need to expand the `buf` dynamically according to the return value of `enif_get_string`.

so there are two options.
1. implement the above consideration
2. Add the todo comment about the above consideration